packages/components/src/components/contract-card/contract-card.scss

```diff
@@ -311,44 +308,38 @@
         }
     }
     &--turbos {
+        .dc-contract-card {
+            &__stake {
+                grid-column: 1/1;
+                grid-row: 1/1;
+                padding: 0.8rem 0;
+            }
+            &__current-price {
+                grid-column: 2/2;
+                grid-row: 1/2;
+                padding: 0.8rem 0;
+            }
+            &__buy-price {
+                grid-column: 1/2;
+                grid-row: 2/2;
+            }
+            &__barrier-level {
+                grid-column: 2/2;
+                grid-row: 2/2;
+            }
+            &__limit-order-info {
+                grid-row: 2/2;
+                grid-column: 2/2;
+            }
+            &__status {
+                align-self: center;
+                justify-self: center;
+                grid-row: 1/3;
+                grid-column: 3/3;
+            }
+        }
         @include mobile {
-            background-color: var(--general-main-1);
             grid-template-columns: 1fr 1fr 1fr;
-            grid-template-areas:
```

Comment:
I deleted `grid-template-areas` property for mobile, because there are no elements, which use `grid-area`, all of them has specified `grid-column` and `grid-row`.